### DISCUSSION TITLE
Using templates instead of Static files for Dockerfiles, etc.

### DISCUSSION THREAD

**Discussion by mhkarimi1383** [tiangolo/uvicorn-gunicorn-docker#208] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
Nothing
```

### Description

Now images are made using static Dockerfiles, Configs, etc.

What if we move to a template engine so we can keep things up to date faster or automatically with ease.
just like some other official images 

### Operating System

Linux

### Operating System Details

ArchLinux

### Python Version

Python 3.12.2

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-docker#-warning-you-probably-dont-need-this-docker-image
